### ISSUE TITLE
rapid fire

### ISSUE THREAD

**Issue by marcel402** [minglich/logitech-pubg#16]
hey brother, dont work rapid fire?

**Comment by dannyfear11** [comment]
No. Use guns on auto. Single or burst won't work.  Is a script not a hack.
It does not know when you are on auto or solo .....It uses only auto guns

On Jun 7, 2018 21:20, "marcel402" <notifications@github.com> wrote:

hey brother, dont work rapid fire?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/minglich/logitech-pubg/issues/16>, or mute the thread
